### PR TITLE
Working With ES6

### PR DESCRIPTION
**Pull request by lorenzoworx**
## Learning objectives

- Use ES6 modules to write modular JavaScript.
- Learn how to use proper ES6 syntax.
- Understand improvements that ES6 brings to JavaScript.
- Use npm as a software package system.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Lorenzo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure to remove the Microverse template logo, from the ReadMe. Optionally, you can replace it with your own logo if you have one.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Lorenzo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦   🏃 

## Highlights 👍  
🎲 Code follows ES6 syntax!
🎲 Date feature properly implemented using Luxon library!
🎲 Good ReadMe!

## Required Changes ♻️
Good job so far, a book detail can be added to the records correctly, however, few problems are showing up in your app, please ensure to fix them:
- [ ] List of books added disappears from the _Book List_ page when the page is refreshed
- [ ] Data still persists in the local storage when they're removed from the _Book List_ page

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- Kindly include appropriate instructions in your ReadMe on how to _run_ your app, under the _Getting Started_ section.
- Kindly endeavor to remove the guide note below the MIT license.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/storage.js [hunk 1]
```diff
@@ -1,12 +1,7 @@
 export default class Storage {
   static getBooks() {
-    let books;
-    if (localStorage.getItem('books') === null) {
-      books = [];
-    } else {
-      books = JSON.parse(localStorage.getItem('books'));//
-    }
-    return books;
+    const books = JSON.parse(localStorage.getItem('books')) || [];
```
Comment (GzimAsani) [inline modules/storage.js:3]: - [ ] After I added a book and went to the book index page I got an error and no books showed up. Kindly fix it
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @lorenzoworx             :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with adding the request from previous reviewer :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/storage.js (hunk 1 above)
Comment (lorenzoworx) [reply modules/storage.js:3]: Hey @GzimAsani  I can't seem to recreate the error that you are facing on my local machine or via the version running on GitHub
 pages

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @lorenzoworx, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Your app is running as expected 
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Please consider:
- [OPTIONAL] consider fixing the console error as pointed out by the previous code reviewer. The error is coming from your `index.js file`. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lorenzoworx** [comment]
@Mihndim2020 Hey, I’m really new to JavaScript any idea on how to fix it cause I don't see it on my page. I’m going to close this pull request but you can text me on slack with any suggestions and further interaction

**Pull request merged**